lib/notifications/notification_server.ex

```diff
@@ -181,9 +181,36 @@ defmodule Notifications.NotificationServer do
     %{status: bridge_status, lowering_time: lowering_time}
   end
 
-  defp broadcast(notification, registry_key) do
+  @chelsea_bridge_route_ids [
```

Comment:
How did this work before? As far as I can tell, everything worked without these values here before - why add them?

Question-behind-the-question: Is there a way to avoid magic values in the code itself?

Non-blocking.

Reply:
This was "magick-ed" here before https://github.com/mbta/skate/blob/e51917e03ea495402166a0df9a5dd45665fae82c/lib/notifications/notification.ex#L166

I'd be open to looking at a post-refactor which cleans this up and consolidates a bit more, as I'm noticing this data is now duplicated between linking the notifications to affected users and broadcasting to users.

Reply:
Thanks for the explanation - it didn't show up when I searched earlier because I searched for `"743"` with the quotes.